frontend/src/components/Uploader/Uploader.js

```diff
@@ -78,4 +75,34 @@ export default class Uploader extends Component {
       </div>
     );
   };
-};
+}
+
+const styles = StyleSheet.create({
+  wrapper: {
+    padding: "0px 20px",
+    border: "2px solid darkgrey",
+    borderRadius: "10px",
+    boxShadow: "0px 0px 8px 0 #ccc",
+    background: "lightgrey",
+    maxWidth: "240px"
+  },
+  well: {
+    margin: "5px auto",
+    // padding: "10px",
+    // paddingBottom: "10px",
+    // marginBottom: "0px"
+  },
+  image_style: { maxWidth: "215px", maxHeight: "215px" },
```

Comment:
Don't like this keep it expanded, you don't gain much if anything from this and it breaks consistency.